subscription-manager.spec

```diff
@@ -177,8 +177,10 @@ Requires: %{name} = %{version}-%{release}
 %if %use_gtk3
 Requires: pygobject3
 Requires: gtk3
+Requires: abattis-cantarell-fonts
```

Comment:
Please use

```specfile
Requires: font(cantarell)
```

instead, as that won't break the openSUSE/SLES12 dependencies from pending work in #1553 (suse package is just `cantarell-fonts`).